sesam.py

```diff
@@ -972,6 +980,24 @@ def authenticate(self):
             login_via_oauth(self.sesam_node,self.args)
         else:
             pass
+    def replace_jinja_variables(self, contents):
+        pattern = r"~t{{@ (?P<variable>[^@}]+) @}}"
```

Comment:
I'm not sure if ~t is some Sesam keyword but I had to transform that part too, together with the jinja variable. Otherwise, the upload process failed with some invalid Json error. The code actually transforms this structure: ~t{{@ some_variable @}} to this: $ENV(some_variable) and vice versa. Any comments?

Reply:
transit-encoding, you can read about it here https://docs.sesam.io/hub/documentation/data-management/entity-data-model.html#extension-types